torch/_dynamo/variables/builder.py

```diff
@@ -1117,6 +1118,13 @@ def wrap_tensor(self, value: torch.Tensor):
             )
         )
 
+        if (
+            isinstance(value, torch.Tensor)
+            and value.is_nested
+            and not isinstance(value, NestedTensor)
+        ):
+            unimplemented("torch.compile does not support C++ NestedTensor")
```

Comment:
for tracking: same nit regarding referencing strided layout instead of C++ NestedTensor